sesman/chansrv/chansrv_fuse.c

```diff
@@ -2810,6 +2818,18 @@ static unsigned int format_user_info(char *dest, unsigned int len,
         g_strncpy(username, uidstr, sizeof(username) - 1);
     }
 
+    if (getenv("DISPLAY") == NULL)
+    {
+        /* if environment variable is not set, set it to empty string */
+        display[0] = '\0';
+        displaynum[0] = '\0';
+    }
+    else
+    {
+        g_strncpy(display, getenv("DISPLAY"), sizeof(display) - 1);
+        g_snprintf(displaynum, sizeof(displaynum), "%d", g_get_display_num_from_display(display));
+    }
+
     return g_format_info_string(dest, len, format, map);
```

Comment:
That should be OK, but `g_get_display_num_from_display()` can return -1, in which case '%d' will also map to -1. This shouldn't happen, as to get here the call has to have succeeded in the past. Thoughts?

Also, we've got an 80-char limit in the [style guide](https://github.com/neutrinolabs/xrdp/wiki/Coding-Style). If you could wrap this line that would be great.